packages/cli-repl/package.json

```diff
@@ -78,13 +78,13 @@
     "@mongosh/types": "0.0.0-dev.0",
     "@segment/analytics-node": "^1.3.0",
     "ansi-escape-sequences": "^5.1.2",
-    "askcharacter": "^1.0.0",
-    "askpassword": "^1.2.4",
+    "askcharacter": "^2.0.4",
+    "askpassword": "^2.0.2",
```

Comment:
Major version bumps because these included changes to the tested (= supported) Node.js versions. I stopped doing these after a bit because it turned out lots of packages needed updates 🙂